app/src/main/java/io/homeassistant/companion/android/settings/qs/views/ManageTilesView.kt

```diff
@@ -0,0 +1,161 @@
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringArrayResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.graphics.drawable.DrawableCompat
+import androidx.core.graphics.drawable.toBitmap
+import androidx.fragment.app.FragmentManager
+import com.maltaisn.icondialog.IconDialog
+import io.homeassistant.companion.android.common.R
+import io.homeassistant.companion.android.database.AppDatabase
+import io.homeassistant.companion.android.database.qs.TileEntity
+import io.homeassistant.companion.android.settings.qs.ManageTilesViewModel
+
+@Composable
+fun ManageTilesView(
+    viewModel: ManageTilesViewModel,
+    iconDialog: IconDialog,
+    childFragment: FragmentManager
+) {
+    val context = LocalContext.current
+    var expandedTile by remember { mutableStateOf(false) }
+    var expandedEntity by remember { mutableStateOf(false) }
+    Column(modifier = Modifier.padding(20.dp)) {
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Text(
+                text = stringResource(R.string.tile_select),
+                fontSize = 15.sp,
+                modifier = Modifier.padding(end = 10.dp)
+            )
+            OutlinedButton(onClick = { expandedTile = true }) {
+                Text(
+                    viewModel.selectedTileName.value
+                )
+            }
+        }
+
+        DropdownMenu(expanded = expandedTile, onDismissRequest = { expandedTile = false }) {
+            val tileNameArray = stringArrayResource(id = io.homeassistant.companion.android.R.array.tile_name)
+            val tileIdArray = stringArrayResource(id = io.homeassistant.companion.android.R.array.tile_ids)
+            for ((tileName, tileId) in tileNameArray.zip(tileIdArray)) {
+                DropdownMenuItem(onClick = {
+                    viewModel.selectedTile.value = tileId
+                    viewModel.selectedTileName.value = tileName
+                    expandedTile = false
+                    if (viewModel.currentTile() != null)
+                        viewModel.updateExistingTileFields()
+                }) {
+                    Text(tileName)
+                }
+            }
+        }
+
+        Divider()
+        TextField(
+            value = viewModel.tileLabel.value ?: "",
+            onValueChange = { viewModel.tileLabel.value = it },
+            label = {
+                Text(
+                    text = stringResource(id = R.string.tile_label),
+                    modifier = Modifier.padding(start = 10.dp, bottom = 10.dp)

```

Comment:
While the padding at the start looks fine if there is no text, when text has been added it is indented and looks out of place. The bottom padding is also useless/ignored by Compose.
![image](https://user-images.githubusercontent.com/8148535/150851705-454c3cc1-252a-4c14-be74-c495f2abc9f5.png)
(also applies to subtitle input)